src/imageLoader/wadors/loadImage.js

```diff
@@ -23,7 +23,7 @@
 
     // TODO: load bulk data items that we might need
 
-    var mediaType = 'multipart/related; type="application/octet-stream"'; // 'image/dicom+jp2';
```

Comment:
So is this a bug in Orthanc, then? I believe the original PR (https://github.com/chafey/cornerstoneWADOImageLoader/issues/54) said that this followed a correction in the DICOM Standard. 

I guess we should report it there instead of removing the quotes here.